src/pmi/src/pmi_wire.c

```diff
@@ -538,3 +538,30 @@ int PMIU_cmd_send(int fd, struct PMIU_cmd *pmicmd)
   fn_fail:
     goto fn_exit;
 }
+
+/*
+ * This function is used to request information from the server and check
+ * that the response uses the expected command name.  On a successful
+ * return from this routine, the response is parsed into the pmi object.
+ * It can be queried for attributes.
+ */
+int PMII_pmi_get_response(int fd, struct PMII_pmi *pmi, const char *expectedCmd)
```

Comment:
nitpick: this commit uses the old name then gets updated in the next commit

Reply:
Thanks, I'll fix it before merging

Reply:
I'll simply squash the two commits together. Both are updating utility functions before actual usage.